ctapipe/tools/stage1.py

```diff
@@ -77,12 +77,17 @@ class Stage1Tool(Tool):
     def setup(self):
 
         # setup components:
-        self.event_source = EventSource(parent=self)
+        self.event_source = EventSource.from_config(parent=self)
 
         datalevels = self.event_source.datalevels
-        if DataLevel.R1 not in datalevels and DataLevel.DL0 not in datalevels:
+        if (
+            DataLevel.R1 not in datalevels
```

Comment:
This is the same but different from the other tool. Create a small method on `EventSource`? E.g. `EventSource.has_datalevel(any_of={...})`